next.config.js

```diff
@@ -2,12 +2,12 @@
 module.exports = {
   reactStrictMode: true,
   images: {
-    domains: ['images.unsplash.com'],
+    domains: ["images.unsplash.com", "googleusercontent.com"],
```

Comment:
A co z avatarami z Facebooka? Czasami też obrazki z jednego serwisu są w różnych domenach, więc też by było trzeba to jakoś obsłużyć

Reply:
Z tego co widzę to maja dwie domeny:

- fbcdn.net
- akmaihd.net

Wiec je dodam i powinno to działać

https://stackoverflow.com/questions/19568411/list-of-facebook-cdn-addresses